janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/JanusGraphConstants.java

```diff
@@ -73,6 +78,11 @@
             if (!v.isEmpty()) b.add(v);
         }
         COMPATIBLE_VERSIONS = b.build();
+        
+        for (String v : props.getProperty("titan.compatible-versions", "").split(",")) {
```

Comment:
Sorry but didn't quite get that, I actually followed the previous implementation which seems to work just fine, can you please elaborate bit better? thanks.